runtime/service_integ_test.go

```diff
@@ -1003,3 +1003,58 @@ func TestDriveMountFails_Isolated(t *testing.T) {
 		assert.Error(t, err, "unexpectedly succeeded in creating a VM with a drive mount under banned path")
 	}
 }
+
+func TestUpdateVMMetadata_Isolated(t *testing.T) {
```

Comment:
I think there would be some value in adding a container to this test that gets the VM metadata too so we can verify that we are able to spawn containers that have access to it too. There's an existing test case that uses `wget` in the alpine container image which could probably be adapted to hit the mmds endpoint instead: https://github.com/firecracker-microvm/firecracker-containerd/blob/38742eeb4fb5d549b31727a57b8c79e88b204b59/runtime/cni_integ_test.go#L112

Reply:
That's a good idea, I'll try to add it.

Reply:
Added this based on the example you linked.